src/Broadway/Domain/Metadata.php

```diff
@@ -43,24 +56,37 @@ public function merge(Metadata $otherMetadata)
     }
 
     /**
-     * {@inheritDoc}
+     * Returns an array with all metadata.
+     *
+     * @return array
      */
-    public function serialize()
+    public function all()
     {
         return $this->values;
     }
 
     /**
-     * Helper method to construct an instance containing the key and value.
+     * Get a specific metadata value based on key.
      *
-     * @param mixed $key
-     * @param mixed $value
+     * @param string $key
      *
-     * @return Metadata
+     * @return mixed
      */
-    public static function kv($key, $value)
+    public function get($key)
```

Comment:
I considered this however, the code of Symfony predates the `?:` operator to my knowledge, I would suggest writing the consumer like this:

```
$valueForX = $metadata->get('x') ?: 'default';
```

Problem gets more serious with fallbacks:
```
$valueForAbcWithFallbackToXyz = $metadata->get('abc') ?: $metadata->get('xyz');
// instead of 
$valueForAbcWithFallbackToXyz = $metadata->get('abc', $metadata->get('xyz'));
```

Let me know what you think, I would happily add the Symfony fallback behaviour if you still prefer it.